core/instantiate.mli

```diff
@@ -1,7 +1,8 @@
 open CommonTypes
 open Utility
 
-type instantiation_maps = (Types.datatype IntMap.t * Types.row IntMap.t * Types.field_spec IntMap.t)
+(* TODO: after Types refactoring, turn this into singular *)
```

Comment:
remove "done" comment

Reply:
this isn't done and the comment still applies